### DISCUSSION TITLE
Exiting group by double-clicking not displayed on deformed mesh

### DISCUSSION THREAD

**Discussion by beeheemooth** [opentoonz/opentoonz#6772] [General]
### Screenshot or Video Reference

![ezgif-1-19f19d4f3e10](https://user-images.githubusercontent.com/25724133/64075422-b856e380-ccc0-11e9-8bc3-3eb4e7737e9d.gif)

### Steps to Reproduce

1. Create several grouped vector shapes at one level and apply the Plastic Tool to it.
2. Since you cannot edit frames from a deformed column, copy them to another column.
3. In this column, select one of the groups and enter it using the "Enter Group" command. However, the remaining groups will be displayed less brightly. Note that everything is displayed in the same way in a deformed column.
4. Exit the group by double-clicking on the canvas. In a deformed column, everything appears as if you are still inside the group.

If you exit the group using the "Exit Group" command, then everything will be displayed correctly in the deformed column.

If you try to double-click to switch between groups, then this also will not be displayed in the deformed column.

### System Information
* **OpenToonz Version**: latest
* **Operating System**: Windows 8.1

**Comment by RodneyBaker** [comment]
I don't know if this will alleviate/workaround your issue but a useful way of editing an image that has Plastic applied (while maintaining keyframed movement!) is to extend the exposure of the image down one (or more) frames so that it is no longer locked by the plastic mesh exposure.  Once edited that temporary exposure can be removed.  Having the image in the same column is generally preferred because any movement outside of the area established by the plastic mesh's mask will cause portions of the image to disappear.  Perhaps that relates to what you are seeing?
